rhel-8/as.po

```diff
@@ -0,0 +1,111 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+#
+# Translators:
+# Nilamdyuti Goswami <ngoswami@redhat.com>, 2013-2014
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-05-26 17:19+0200\n"
+"PO-Revision-Date: 2015-01-14 06:43+0000\n"
+"Last-Translator: Copied by Zanata <copied-by-zanata@zanata.org>\n"
+"Language-Team: Assamese (http://www.transifex.com/projects/p/initial-setup/language/as/)\n"
```

Comment:
Is the URL still valid?

Reply:
That is not my and it will be probably replaced by Weblate after import.

Reply:
Wow, it that was still there - but yeah, that should get clobbered by Weblate once it starts interacting with the git repo.